lib/blueprinter/helpers/base_helpers.rb

```diff
@@ -44,7 +44,15 @@ def inherited(subclass)
       def object_to_hash(object, view_name:, local_options:)
         result_hash = view_collection.fields_for(view_name).each_with_object({}) do |field, hash|
           next if field.skip?(field.name, object, local_options)
-          hash[field.name] = field.extract(object, local_options)
+
+          begin
```

Comment:
As this is a rather non-standard practice in popular codebases, I would also add a comment here stating that the purpose of this is to wrap all errors under a `BlueprinterError` so it makes it clearer for people non-familiar with that kind of pattern.